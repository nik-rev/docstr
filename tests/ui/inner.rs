fn main() {
    docstr::docstr!(
        //! this
        //! is not
        //! supported
    );
}
