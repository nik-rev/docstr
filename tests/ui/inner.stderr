error: Inner doc comments `//! ...` are not supported. Please use `/// ...`
 --> tests/ui/inner.rs:3:9
  |
3 |         //! this
  |         ^^^^^^^^

error: Inner doc comments `//! ...` are not supported. Please use `/// ...`
 --> tests/ui/inner.rs:5:9
  |
5 |         //! supported
  |         ^^^^^^^^^^^^^
